### PR TITLE
fix: cal video issues

### LINKED ISSUE #11136: Cal video meeting urls error in organizations
1) Create a organization and a event type with cal video as location
2) book a meeting and try to open the meeting url

image

### PR DESCRIPTION
**Pull request by Udit-takkar**
Fixes: https://github.com/calcom/cal.com/issues/11136

1) Hydration error
2) Access cal video meeting link 

Before:- 
image

After:-

Screenshot 2023-11-27 at 5 15 46 PM

### CODE REVIEW

File: apps/web/server.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+app.prepare().then(() => {
+  createServer(httpsOptions, async (req, res) => {
+    const parsedUrl = parse(req.url, true);
+    await handle(req, res, parsedUrl);
+  }).listen(port, (err) => {
+    if (err) throw err;
+    console.log(`ready - started server on url: https://app.cal.local:${port}`);
+  });
+});
```
Comment (Udit-takkar) [inline apps/web/server.js:23]: This script is for serving nextjs server over https for testing organization's cal video meeting links

File: apps/web/server.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  cert: fs.readFileSync("./app.cal.local.pem"),
+};
+
+app.prepare().then(() => {
+  createServer(httpsOptions, async (req, res) => {
+    const parsedUrl = parse(req.url, true);
+    await handle(req, res, parsedUrl);
+  }).listen(port, (err) => {
+    if (err) throw err;
+    console.log(`ready - started server on url: https://app.cal.local:${port}`);
```
Comment (PeerRich) [inline apps/web/server.js:21]: should this be hardcoded like this? are some people not using other local domains?

**Review by PeerRich — APPROVED** [review]
looks good, need @hariombalhara for final review for org changes

**Review by PeerRich — DISMISSED** [review]
looks good, need @hariombalhara for final review for org changes

File: apps/web/server.js (hunk 2 above)
Comment (Udit-takkar) [reply apps/web/server.js:21]: This script is only for development and we have mentioned app.cal.local everywhere in readme

**Review by zomars — CHANGES_REQUESTED** [review]
Let's keep this PR with only the proposed fix.

File: apps/web/server.js (hunk 2 above)
Comment (Udit-takkar) [reply apps/web/server.js:21]: Screenshot 2023-11-27 at 10 39 09 PM

The issue is that Browsers do not allow camera/mic access on any non-HTTPS hosts except for localhost. (Which i confirmed from daily.co team).

so we need script this to run https://app.cal.local:3000/

Comment (Udit-takkar) [reply apps/web/server.js]: Ok I have removed the file and added more instructions in readme.

**Review by zomars — APPROVED** [review]
Code now LGTM. Waiting for checks

**Pull request merged**